11-http/api/ctrls/users.go

```diff
@@ -99,6 +101,7 @@ func (uc usersCtrl) Create(c *app.Context) {
 	c.Params = map[string]string{"id": u.UserID}
 	uc.Retrieve(c)
 
+	// this log message will be wrong if Retrieve fails
```

Comment:
The current logging is a bit oblivous in a few places, and will report success even after a failure occurred, potentially masking the presence of a failure.